src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -827,14 +825,17 @@ public Locomotive longPressSwipeLong(SwipeElementDirection direction, WebElement
         return longPressSwipe(direction, webElement, SWIPE_DISTANCE_LONG, 0);
     }
 
+    @Deprecated
     public Locomotive longPressSwipeLong(SwipeElementDirection direction, String id, int swipeDurationInMillis) {
```

Comment:
All deprecated methods really should have some javadoc on why they are deprecated and what you should do about it.

Reply:
I think @aaron-goff deprecated these and I dunno the reasoning :\ I just added these because the other method had the annotation

Reply:
🤔 I'm not sure that I deprecated these? I think these ones are the ones that shouldn't be deprecated?

Reply:
Ohh I see -- you deprecated the one that takes in a webelement -- i'll add a comment

Reply:
Fixed in ef29718